assets/src/components/reader-theme-selection/index.js

```diff
@@ -56,17 +76,25 @@ export function ReaderThemeSelection( { disableCurrentlyActiveTheme = false } )
 					__( 'Select the theme template for mobile visitors', 'amp' )
 				}
 			</p>
+			{ activeTheme && hideCurrentlyActiveTheme && (
+				<AMPNotice>
+					{
+						sprintf(
+							/* translators: placeholder is the name of a WordPress theme. */
+							__( 'Your active theme "%s" is not available as a reader theme. If you wish to use it, Transitional mode may be the best option for you', 'amp' ),
```

Comment:
I'm not a grammar nazi I  swear :laughing: 

```suggestion
							__( 'Your active theme "%s" is not available as a reader theme. If you wish to use it, Transitional mode may be the best option for you.', 'amp' ),
```